internal/server/v1/page/golden/place_page/asm.Demographics.json

```diff
@@ -180,15 +180,18 @@
             "2008": 708173810,
             "2009": 693852712,
             "2010": 679206337,
-            "2011": 665196496,
-            "2012": 651507733,
-            "2013": 637792141,
-            "2014": 624030741,
-            "2015": 610192900,
-            "2016": 596465626,
-            "2017": 582840458,
-            "2018": 568902350,
-            "2019": 554781038
+            "2011": 665644371,
```

Comment:
These are coming from some OECD import (not WDI). I am not sure if this is expected?